localstack/services/ses/ses_listener.py

```diff
@@ -0,0 +1,29 @@
+import re
+import xmltodict
+from localstack.utils.common import to_str
+from localstack.utils.aws.aws_responses import make_error
+from localstack.services.generic_proxy import ProxyListener
+
+
+class ProxyListenerSES(ProxyListener):
+    def forward_request(self, method, path, data, headers):
+        return True
+
+    def return_response(self, method, path, data, headers, response):
+        has_xmlns = self.response_has_xmlns(to_str(response.content))
+        if not has_xmlns and response.status_code >= 400:
+            content_dict = xmltodict.parse(to_str(response.content))
+            error_description = content_dict['ErrorResponse']['Errors']['Error']
```

Comment:
Can we assume that the error details are always present - I'd try to make the code a bit more resilient to different response formats (to avoid `KeyError`):
```
error_description = content_dict.get('ErrorResponse', {}).get('Errors', {}).get('Error', {})
code = error_description.get('Code')
message = error_description.get('Message')
```

Reply:
mmmm I was thinking that maybe instead of rebuilding the whole xml message to add the xmlns attribute, maybe the code should just add it to the string. What do you think?